plugin/restore/mysql/mysql.go

```diff
@@ -378,6 +380,10 @@ func (r *Restore) SwapPITRDatabase(ctx context.Context, database string, suffixT
 		return pitrDatabaseName, pitrOldDatabase, nil
 	}
 
+	if _, err := db.ExecContext(ctx, "SET sql_log_bin=OFF"); err != nil {
```

Comment:
Let's put it after L350, because we may execute a `CREATE DATABASE` at L360.

Reply:
My fault.